src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/PostsListPage.java

```diff
@@ -27,6 +27,7 @@
 public class PostsListPage extends WikiBasePageObject implements AvailablePage {
 
   private static final String PATH = "/d/f/";
+  public static final String FULL_PATH = "/d/f?sort=latest";
```

Comment:
I'm I correct to assume that this is used when trying to establish if we're done with redirecting to the desired page? I wonder if there's a way to do that without having to rely on this.

Reply:
Yes - it is used in that context, but no - we don't rely on this (we still wait for whole page to load!). The problem was trailing `/` which is present in `PATH` but never in our urls simply because we redirect `/d/f/` to `/d/f`. I just changed original `PATH` not to contain trailing `/` and removed `FULL_PATH`